### PR TITLE
add rear view radar (RDR protocol) support

### PR DESCRIPTION
**Pull request by tensorturtle**
Hi @zacharyedwardbull , please take a look.

Some questions:
+ I don't know how your documentation is generated, so I just copied the formats in other modules
+ When the radar unit boots up, the first 10 or so packets it sends are 'malformed'. Currently I print the error, return None, and continue. What is your opinion on how this should be handled?

### CODE REVIEW

File: pycycling/rear_view_radar.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+Other models which are expected to support RDR service are:
+    Garmin RTR515, RTR516 (German market version), and RCT715
+    Bryton Gardia R300
+    Magene L508
+
+Example
+=======
+This example prints radar information broadcast from the Bluetooth device to the console. Please see also
+information on :ref:`obtaining the Bluetooth address of your device <obtaining_device_address>`.
+.. literalinclude:: ../examples/cycling_power_service_example.py
```
Comment (zacharyedwardbull) [inline pycycling/rear_view_radar.py]: Should this be `rear_view_radar_example.py`?

**Review by zacharyedwardbull — COMMENTED** [review]
Thanks for the PR, looks good. I only noticed one thing, and I think there are a few small things that the linter picked up. Once that's sorted I'll merge.

File: pycycling/rear_view_radar.py (hunk 1 above)
Comment (tensorturtle) [reply pycycling/rear_view_radar.py]: The other way around- the docstring needs to go in `example/rear_view_radar_example.py`.

Comment (zacharyedwardbull) [reply pycycling/rear_view_radar.py]: Sorry I mean we should replace `../examples/cycling_power_service_example.py` here with `../examples/rear_view_radar_example.py` so we include the correct example in the documentation.

Comment (tensorturtle) [reply pycycling/rear_view_radar.py]: Ah, yes indeed.

**Comment by tensorturtle** [comment]
Thank you @zacharyedwardbull ! This is actually my first ever public PR, so I'm happy I could contribute.

**Comment by zacharyedwardbull** [comment]
Thank you and congrats! 🎉

**Pull request merged**